<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QWidget" name="SelectionWindow" native="true">
     <property name="font">
      <font>
       <family>Open Sans</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="BacktoRes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>11</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 0, 127);
color: rgb(255, 255, 255);
border-radius:10;</string>
        </property>
        <property name="text">
         <string>BACK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Done">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>11</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 85, 255);
color: rgb(255, 255, 255);
border-radius:10;</string>
        </property>
        <property name="text">
         <string>DONE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="AddOfficer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>11</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
border-radius:10;</string>
        </property>
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="Title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(129,199,132);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SELECT AUTHORIZATIONS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
